OsmAnd/src/net/osmand/plus/quickaction/SelectMapViewQuickActionsBottomSheet.java

```diff
@@ -106,6 +110,14 @@ public void createMenuItems(Bundle savedInstanceState) {
 		populateItemsList();
 	}
 
+	@LayoutRes
+	private int getLayoutId() {
+		if (action instanceof ChangeMapOrientationAction) {
+			return R.layout.bottom_sheet_item_with_descr_and_radio_btn;
```

Comment:
Why do we use layout with description & then hide it via AndroidUiHelper.updateVisibility(descriptionTv, false); ?